__test__/pages/bib/bibPage.test.tsx

```diff
@@ -168,11 +168,41 @@ describe("Bib Page Item Table", () => {
     await userEvent.click(yearFilter.querySelector("button[type='submit']"))
 
     expect(mockRouter.asPath).toBe("/bib/pb5579193?item_date=2005")
+  })
+
+  it("clears the year filter when the year tag is clicked", async () => {
+    const yearTag = screen.queryByText("Year > 2005").closest("button")
+    await userEvent.click(yearTag)
+
+    expect(mockRouter.asPath).toBe("/bib/pb5579193")
```

Comment:
Hmm I just noticed that these tests depend on the previous test having run and pass. If this test ever gets commented out, skipped, or deleted, the next test will fail. Not something to fix now but tests should be as isolated as possible, unless if there is a need to make them sequential.

Reply:
I agree. I noticed that too but didn't fix since it's out of scope. Will open a ticket to fix this.